app/views/feedback/steps/completed.html.erb

```diff
@@ -0,0 +1,8 @@
+<% @hide_page_helpful_question = true %>
+
+<section>
+  <h1 class="heading-l heading--box-green feedback-thankyou">Thank you for your feedback.</h1>
```

Comment:
```suggestion
  <h1 class="heading-l heading--box-green feedback-thankyou">Thank you for your feedback</h1>
```

Reply:
Agree - there shouldn't be a full-stop in an h1 for consistency

@StackedPancakez Can we remove as is or does anything else depend on this and thus also need changing?